packages/cubejs-client-core/index.d.ts

```diff
@@ -252,9 +252,13 @@ declare module '@cubejs-client/core' {
      */
     y?: string[];
     /**
-     * If `true` missing dates on the time dimensions will be filled with `0` for all measures.Note: the `fillMissingDates` option set to `true` will override any **order** applied to the query
+     * If `true` missing dates on the time dimensions will be filled with fillWithValue or `0` by default for all measures.Note: the `fillMissingDates` option set to `true` will override any **order** applied to the query
      */
     fillMissingDates?: boolean | null;
+    /**
+     * Value to autofill all the missing date's measure.
+     */
+    fillWithValue?: string | null;
```

Comment:
Thnx for adding another test! 
As number is still a valid type for this, let's updat the typedef too:
```suggestion
    fillWithValue?: string | number | null;
```